anyway/request_params.py

```diff
@@ -95,7 +95,7 @@ def get_request_params_from_request_values(vals: dict) -> Optional[RequestParams
     years_ago = vals.get("years_ago", BE_CONST.DEFAULT_NUMBER_OF_YEARS_AGO)
     lang = vals.get("lang", "he")
     location_text = location["text"]
-    gps = location["gps"]
+    gps = {"lat": news_flash_obj.lat, "lon": news_flash_obj.lon}
```

Comment:
Hi,

1. In this location `news_flash_obj` may be `None`
2. This change is not needed because if `news_flash_obj` is not None, these values are already placed in the `gps` dict by `get_location_from_news_flash_or_request_values()`.